approve.py

```diff
@@ -32,14 +32,17 @@
     callink_oid: {callink_oid}
     signatory: \n\
     email: {email}
+    dry_run: FALSE
```

Comment:
why uppercase? we use lowercase booleans in the rest of our yaml everywhere else